src/pages/hyungminJeon/Login/Login.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import "./Login.scss";
+
+const Login = () => {
+  const navigate = useNavigate();
+  const [userInfo, setUserInfo] = useState({
+    userId: "",
+    userPw: "",
+  });
+
+  const goToMain = () => {
+    if (isActive) {
+      navigate("/hyungminjeon-main");
+    } else {
+      alert("아이디를 확인해주세요.");
+    }
+  };
+
+  const goToNoLogin = () => {
+    alert("로그인을 해주세요.");
+  };
+
+  const handleInput = (event) => {
+    console.log(event);
+    const { value, id } = event.target;
+    setUserInfo({ ...userInfo, [id]: value });
+  };
+
+  const isActive = userInfo.userId.includes("@") && userInfo.userPw.length >= 5;
+
+  return (
+    <div className="login">
+      <div className="box_border">
+        <button className="logo_btn" onClick={goToNoLogin}>
```

Comment:
```suggestion
      <div className="boxBorder">
        <button className="logoBtn" onClick={goToNoLogin}>
```
js의 기본적인 단어작성 컨벤션은 `camelCase`입니다!! className도 예외는 아니겠죠??

태그에 className을 주신부분에 camelCase가 아닌 방식으로 작성된 모든부분을 꼭 수정해주세요!